static/lang/re-en.json

```diff
@@ -526,6 +526,9 @@
                     }
                 }
             },
+            "Xulgath": {
+                "StenchSuppression": "Stench Suppression"
```

Comment:
```suggestion
```
If it's just the name of the ability it's on then we don't need a localization for it.